lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1086,27 +1086,38 @@ def restart_sctpd(self, wait_time=30):
                 "/usr/local/bin/config_stateless_agw.py sctpd_pre';"
                 "docker start sctpd mobilityd pipelined sessiond oai_mme",
             )
-        self.wait_for_restart_to_finish(wait_time)
 
-    @staticmethod
-    def wait_for_restart_to_finish(wait_time):
-        for j in range(wait_time):
-            print(
-                f"Waiting for {wait_time - j} seconds for restart to complete",
-            )
-            time.sleep(1)
+    def wait_for_restart_to_finish(self, wait_time):
+        """wait for started services to become active or until timeout
+
+        Args:
+            services: (list[str]) list of services to check
+            wait_time: (int) max time to wait for services to become active
+        """
```

Comment:
We need to add a generic comment here mentioning the wait duration, so that the user is aware of the sleep:
print(f"Waiting for maximum {wait_time} seconds for restart to complete")